presto-hive/src/test/java/com/facebook/presto/hive/TestHiveMetadataFileFormatEncryptionSettings.java

```diff
@@ -125,13 +143,15 @@ public void tearDown()
         }
     }
 
-    private static ConnectorTableMetadata getConnectorTableMetadata(String tableName, Map<String, Object> tableProperties)
+    private static ConnectorTableMetadata getConnectorTableMetadata(String tableName, Map<String, Object> tableProperties, boolean isPartitioned)
```

Comment:
Maybe I missed it, but I think none of these tests are using unpartitioned tables. Maybe add a test for that?

Reply:
There were already `CTAS` tests to which I amended the write side of the tests. That said, you are correct that I need to add tests for `CREATE` and `INSERT INTO` - let me see if the code supports that or not :)